File: src/main/java/no/nav/syfo/domain/Innsending.java
package no.nav.syfo.domain;

import lombok.Builder;
import lombok.Getter;
import lombok.NonNull;
import lombok.Value;

import java.time.LocalDate;

@Value
@Builder(toBuilder = true)
@Getter
public class Innsending {
    @NonNull String innsendingsId;
    @NonNull String ressursId;
    String aktorId;
    String saksId;
    String journalpostId;
    String oppgaveId;
    LocalDate behandlet;
}

File: src/main/kotlin/no/nav/syfo/domain/Innsending.kt
package no.nav.syfo.domain

import java.time.LocalDate

data class Innsending(
    val id: String,
    val sykepengesoknadId: String,
    val journalpostId: String? = null,
    val oppgaveId: String? = null,
    val behandlet: LocalDate? = null,
    val soknadFom: LocalDate? = null,
    val soknadTom: LocalDate? = null
)
